docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+# [TAP 7.1] Solution Design Hausanschluss Bundle Konzept
+
+# 1 Business Case
+
+## 1.1 Executive Summary
+
+The EDC "Hausanschluss" Bundle aims to simplify the installation of the enablement services Eclipse Dataspace
+Connector (EDC), Digital Twin Registry (DTR), and their supporting components. By developing a flexible, parameterizable
+installation package, the project seeks to reduce deployment complexity and lower barriers to participation in the
+Catena‑X data space. This bundle will enable independent component updates and support a modular "bring your own"
+approach, ultimately fostering a more efficient and accessible ecosystem for diverse participants—such as testers,
+external developers, and small and medium-sized enterprises.
+
+## 1.2 Business Need
+
+Current deployment methods for data space integration are complex and require significant manual effort. This PoC
+addresses the need to:
+
+- Reduce Complexity: Simplify the deployment topology by bundling components into capabilities.
+- Automate Deployment: Provide an automated installation package using Kubernetes and HELM.
+- Reduce Configuration Efforts: Streamline and reduce the number of configuration steps required for integration.
+- Extend Documentation: Provide a detailed documentation tailored to the target groups.
+- Enable Extension: Design the system to allow the seamless addition of new components or capabilities.
+
+## 1.3 Objectives
+
+- Simplify Deployment and Maintenance: Develop an installation package that minimizes manual configuration through
+  automation.
+- Ensure Modularity: Structure the bundle so that components can be independently installed, updated, or replaced
+  without
+  affecting other services.
+- Facilitate Flexibility: Implement a "bring your own" model for key components, such as EDC, PostgreSQL, and HashiCorp
+  Vault, to allow tailored deployments that meet specific needs.
+- Increase Accessibility: Lower entry barriers for new participants by reducing technical complexity and operational
+  dependencies.
+- Scalability: Structure the bundle so that it can be deployed multiple times.
+
+## 1.4 Expected Deliverables
+
+- PoC Documentation: A concept and architecture overview highlighting the modular design.
+- Modular Installation Package: A set of HELM charts for deploying the enablement service bundles:
+    - EDC Bundle: Including the Eclipse Dataspace Connector, EDC PostgreSQL, and HashiCorp Vault.
```

Comment:
Are we willing to create a HELM for the HashiCorp Vault

Reply:
no, there is no need for that. The table was not migrated correctly which lead to some missleading content. This is now fixed